src/drt/src/dr/FlexDR.cpp

```diff
@@ -2328,6 +2328,148 @@ static std::vector<FlexDR::SearchRepairArgs> strategy()
           /* 63 */ {7, -6, 64, shapeCost * 64, MARKERCOST * 16, 0, false}};
 }
 
+void FlexDR::reportGuideCoverage()
+{
+  using namespace boost::polygon::operators;
+
+  ofstream file(GUIDE_REPORT_FILE);
```

Comment:
Its odd to split the writing of the file around the computation.  Just move this after the computation is done.